From 6a743c26db275158fd4c90533c37e4559eef807e Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Fri, 28 Jan 2022 08:21:40 +0200
Subject: [PATCH] [SW-69188] gaudi3: add empty function for ram

RAM functionality is not required and supported in gaudi3. This patch
add dummy function for RAM image load.

Change-Id: Ib4dd0675ff32a25315a030cf872927e77d99db0b
---

diff --git a/app/arcboot_rom.c b/app/arcboot_rom.c
index a4afe00..4630a1e 100644
--- a/app/arcboot_rom.c
+++ b/app/arcboot_rom.c
@@ -425,7 +425,7 @@
 	return ARCBOOT_BRINGUP_OK;
 }
 
-static void arcboot_rom_bringup_arcboot_ram(void)
+__weak void arcboot_rom_bringup_arcboot_ram(void)
 {
 
 	arcboot_rom_status_update(PROGRESS_BRINGUP_RAM_PRI);
diff --git a/arch/gaudi3/arc/common.c b/arch/gaudi3/arc/common.c
index 20a5db3..ac3f02d 100644
--- a/arch/gaudi3/arc/common.c
+++ b/arch/gaudi3/arc/common.c
@@ -12,6 +12,8 @@
 
 #include <d0_psoc_global_conf_masks.h>
 
+void arcboot_rom_bringup_arcboot_ram(void) {}
+
 void arcboot_rom_clear_sw_status(void)
 {
 	/* As per Architecture recommendation, clear status reg */
